### DISCUSSION TITLE
Short Piece Reviewing PubPub

### DISCUSSION THREAD

**Discussion by jeffpooley** [knowledgefutures/pubpub#2168] [General]
Hi all,

I thought I’d share this [short piece](https://elephantinthelab.org/the-case-for-pubpub/) I wrote making the case for PubPub (as an admiring adopter and KFG member). Any reactions more than welcome.

**Comment by gabestein** [comment]
Jeff — thanks so much for writing this. It's an amazing vote of confidence, and we're lucky to have you as a user and supporter. We're looking forward to more to come!
